src/bin/git-stack/stack.rs

```diff
@@ -1453,8 +1453,8 @@ struct RenderNode<'r> {
 }
 
 const GLYPHS: termtree::GlyphPalette = termtree::GlyphPalette {
-    middle_item: "⌽",
-    last_item: "⌽",
+    middle_item: "○",
```

Comment:
To be even closer to `tig` this could be `∙` instead

https://github.com/jonas/tig/blob/06a1b89d98f32e9ac2d81b94515b200b0acb7dbb/test/graph/18-tig-test#L30-L34